grpc/server/pom.xml

```diff
@@ -152,6 +153,12 @@
             <artifactId>bedrock-testing-support</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-tcnative-boringssl-static</artifactId>
+            <version>${version.netty.tcnative}</version>
```

Comment:
The version will be picked up from the very top level `pom.xml` so the `<version>` element should not be specified here.